src/main/java/com/google/api/generator/gapic/utils/SampleCodeJavaFormatter.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.utils;
+
+import com.google.api.generator.engine.ast.Statement;
+import com.google.api.generator.engine.writer.JavaFormatter;
+import com.google.api.generator.engine.writer.JavaWriterVisitor;
+import java.util.List;
+
+public final class SampleCodeJavaFormatter {
+
+  private SampleCodeJavaFormatter() {}
+
+  private static final String RIGHT_BRACE = "}";
+  private static final String LEFT_BRACE = "{";
+  private static final String NEWLINE = "\n";
+
+  private static final String FAKE_CLASS_TITLE =
+      String.format("public class FakeClass %s%s", LEFT_BRACE, NEWLINE);
+  private static final String FAKE_METHOD_TITLE =
+      String.format("void fakeMethod() %s%s", LEFT_BRACE, NEWLINE);
+  private static final String FAKE_METHOD_CLOSE = String.format("%s%s", RIGHT_BRACE, NEWLINE);
+  private static final String FAKE_CLASS_CLOSE = String.format("%s", RIGHT_BRACE);
+
+  /**
+   * format utilize google-java-format to format the sample code statements where these are wrapped
```

Comment:
In general, the class comment should only describe a high-level summary of the expected behavior, and we should avoid mentioning implementation details here.

Reply:
My purpose is to explain the why we need to wrap the code into a class for google-java-format. I moved the comment inside of `format` method then. 
What do you suggest on this comment? Do you think it is necessary? Or maybe we just simply remove the comment. LMK what you think. Thanks

Reply:
Moving the comment into the method SGTM.